src/components/SignInPage.js

```diff
@@ -0,0 +1,25 @@
+import "./SignInPage.css";
+
+export default function SignInPage() {
+  return (
+    <div className="app-grid">
+      <header>
+        <p className="signIn-headline">name</p>
+      </header>
+      <main>
+        <div className="signIn-image">
+          <input
```

Comment:
This will work. If you would like the user to be able to submit the form by just pressing enter rather than the button, you'll have to put it into a form tag (I know, less of a mobile issue, more of a desktop thing)